### PR TITLE
Add export_dir_path to unload_and_copy

### PR DESCRIPTION
**Pull request by lisovskey**
I've tried to add a possibility to choose directory for downloaded files in `unload_and_copy` method
Not tested

### CODE REVIEW

**Comment by fdosani** [comment]
thanks for the PR @lisovskey. Would you be able to provide some more context on what feature you are looking or hoping to implement here? Just taking a quick look at this and wondering if rather than having a new input we could reuse `export_path`?  Right now it is just writing those s3 files to the `cwd`, but I guess you would like to pick the spot?

**Comment by lisovskey** [comment]
You’re right, I’d like to choose a folder for s3 files. I think two parameters are more explicit and usability is more obvious. But reusing `export_path` is also an option. For example, if directory path is passed files will be written to this path, but in case of file path files will appear at the directory of the path and then will concatenate into single file

**Comment by fdosani** [comment]
@jborchma FYI, just wanted to loop you in.

**Comment by fdosani** [comment]
@lisovskey First of thanks very much to contributing this PR. I think there are maybe a couple of ways we can go here.

1. Since the data is being unloaded to s3 a user can copy down the data once the `unload_and_copy ` is completed
2. We download it to the specific location the end user wants (your proposal)
3. OR just to the same location as `export_path`.

@jborchma @elzzhu @dasjyao @theianrobertson  any thoughts on this? I'm ok with 2. 👍

**Comment by jborchma** [comment]
2. seems ok to me too

**Comment by dasjyao** [comment]
2 looks good to me as well.

File: locopy/redshift.py [hunk 1]
```diff
@@ -337,9 +338,13 @@ def unload_and_copy(
             The AWS S3 folder of the bucket where the data from the query will
             be unloaded. Defaults to ``None``. Please note that you must follow
             the ``/`` convention when using subfolders.
+        
+        export_dir_path : str, optional
```
[the remark this answers is not in the document]
Comment (lisovskey) [reply locopy/redshift.py:342]: @fdosani make sense, yes

File: tests/test_redshift.py [hunk 2]
```diff
@@ -658,6 +658,7 @@ def reset_mocks():
             query="query",
             s3_bucket="s3_bucket",
             s3_folder=None,
+            export_dir_path=None,
```
[the remark this answers is not in the document]
Comment (lisovskey) [reply tests/test_redshift.py:661]: Of course, I've added you as collaborator and filled out CLA

[the remark this answers is not in the document]
Comment (fdosani) [reply tests/test_redshift.py:661]: Thanks! Sorry for the delay here. I'm going to take some time tomorrow and work on this for the integration tests.

[the remark this answers is not in the document]
Comment (fdosani) [reply tests/test_redshift.py:661]: Ok, I've made a PR on your dev fork. If you want to take a look and merge it in we can continue the review here.

**Comment by fdosani** [comment]
@jborchma @dasjyao Would you mind chiming in with a review here. I added a few things to @lisovskey fork.

File: tests/test_redshift.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (lisovskey) [reply tests/test_redshift.py]: Yep, done

**Pull request merged**